newIDE/app/src/AssetStore/AssetStoreNavigator.js

```diff
@@ -178,16 +200,34 @@ export const useNavigation = (): NavigationState => {
           ],
         }));
       },
-      openPackPage: (assetPack: PublicAssetPack | PrivateAssetPack) => {
+      openPackPage: ({
+        assetPack,
+        previousSearchText,
```

Comment:
Is this new argument really needed (just a challenge, since I see it was not needed before)

Reply:
Yes, before we were not saving the search text as we never needed to rehydrate it from pages.
Now when clicking on a pack via a first search, the search text is cleared and we need to be able to go back to that previous search when clicking back